src/NServiceBus.Core.Tests/Msmq/MsmqTestsBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Core.Tests.Msmq
+{
+    using System.Messaging;
+
+    public class MsmqTestsBase
```

Comment:
@danielmarbach I agree, but I was looking for a test that did a similar thing, and this time I *didn't* want to copypaste all the setup code to my own, so I extracted this base class. Could be better, I agree.